gbpservice/nfp/configurator/agents/agent_base.py

```diff
@@ -0,0 +1,181 @@
+from oslo_log import log as logging
```

Comment:
Is this a module for configurator. Should this be under configurator/modules dir?